.gitignore

```diff
@@ -2,3 +2,16 @@ node_modules
 .idea
 bin/
 npm-debug.log
+
+# OS generated files #
```

Comment:
imho maybe these should be in your personal global gitignore and not in the repo